impeller/entity/contents/framebuffer_blend_context.h

```diff
@@ -0,0 +1,295 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
```

Comment:
Now that there are no compile time checks I could just move this back into content_context.h